config/initializers/api_keys.rb

```diff
@@ -1,4 +1,2 @@
-s3 = Aws::S3::Client.new
-resp = s3.get_object( bucket: ENV['S3_BUCKET_NAME'], key: ENV['API_KEYS_PATH'] )
-API_KEYS = YAML::load(resp.body.read)
-Rails.logger.info { "Successfully loaded keys from S3" }
+config_file_path = File.join(ENV['SHARED_CONFIG_ROOT'] || "#{Rails.root}/config", "api-keys.yml")
```

Comment:
Please be very careful about making changes like this!! This is going to be break the app, so I'll revert it.